package.json

```diff
@@ -4,6 +4,10 @@
   "description": "Ledger Nano S wallet provider for the Web3 ProviderEngine",
   "main": "lib/index.js",
   "scripts": {
+    "ganche": "./test/ganache-cli.sh",
+    "test-setup": "./node_modules/.bin/babel-node ./test/setup.js",
```

Comment:
you can use just `babel-node` in scripts yarn makes sure that your PATH includes `./node_modules/.bin`.

Reply:
And its even stated in npm doc - thank you for this note :)